analyze.sh

```
@@ -0,0 +1,166 @@
+# Run Android Lint (built into Android Gradle)
+echo "  Running Android Lint..."
+
+# Check for existing lint results (so we can show them if available)
+LINT_TXT_REPORT="build/reports/lint-results-debug.txt"
+LINT_XML_REPORT="build/reports/lint-results-debug.xml"
+
+# Run lint (will be quick if nothing changed)
+if ./gradlew lint -q 2>&1 | grep -q "BUILD SUCCESSFUL\|UP-TO-DATE"; then
+    LINT_SUCCESS=true
+else
+    LINT_SUCCESS=false
```

Shellcheck (0.11.0)

[warning] LINT_SUCCESS appears unused. Verify use (or export if used externally).

(SC2034)